internal/plugins/golang/v2/api.go

```diff
@@ -19,7 +19,7 @@ import (
 	"sigs.k8s.io/kubebuilder/pkg/model/config"
 	"sigs.k8s.io/kubebuilder/pkg/plugin"
 
-	"github.com/operator-framework/operator-sdk/internal/plugins/sdk"
+	"github.com/operator-framework/operator-sdk/internal/pluginutil/sdk"
```

Comment:
Why bother changing the import path at all?

Reply:
See : https://github.com/operator-framework/operator-sdk/pull/3349#discussion_r452303533

Reply:
In that case:
```suggestion
	"github.com/operator-framework/operator-sdk/internal/plugin/util"
```

Reply:
The import is required because of ` sdk.RunCreateAPI(p.config)` line 57 which is in the `"github.com/operator-framework/operator-sdk/internal/pluginutil/sdk"`

Reply:
I'm suggesting you change the package path too.

Reply:
The name was a suggestion made by @joelanford already https://github.com/operator-framework/operator-sdk/pull/3349#discussion_r452303533 and I do not think that it should be a blocker.

Reply:
I would really prefer my suggestion over having `sdk` in the path.

Reply:
I will change for `internal/util/plugins`..